### DISCUSSION TITLE
tts 好久没更新了，要不要来个C++版的，最好再做个VITS模型的

### DISCUSSION THREAD

**Discussion by cgisky1980** [RapidAI/RapidTTS#8] [Ideas]

**Comment by SWHL** [comment]
欢迎贡献

**Comment by cgisky1980** [comment]
> 欢迎贡献

https://github.com/rhasspy/larynx2/issues/9      这个 在训练中文了

**Comment by SWHL** [comment]
期待中... ...

**Comment by MagiciSource** [comment]
同期待，最好是中英文混合支持的

**Comment by shen2009fei** [comment]
哈哈ASR已经开始使用了 期待C++版本的 TTS😚

**Comment by shen2009fei** [comment]
sambert-hifigan 貌似声音效果很好

**Comment by shen2009fei** [comment]
看了下论文貌似VITS，比之前的技术更先进，实时性更好，更接近人类声音。
貌似有个C++的repo了，不过不支持train 和 英文。

https://github.com/huakunyang/SummerTTS

**Comment by lucasjinreal** [comment]
https://github.com/lucasjinreal/vits_cpp

**Comment by SWHL** [comment]
这个[SambertHifigan语音合成-中文-多人预训练-16k
](https://www.modelscope.cn/models/speech_tts/speech_sambert-hifigan_tts_zh-cn_multisp_pretrain_16k/summary)（支持中英和数字混合生成），就是没有onnx的

**Comment by shen2009fei** [comment]
这个基于VITS的 也不错，目前全网发现性能最高的repo了。 就是依赖太恶心了。
https://github.com/rhasspy/piper

**Comment by cgisky1980** [comment]
> 这个基于VITS的 也不错，目前全网发现性能最高的repo了。 就是依赖太恶心了。 https://github.com/rhasspy/piper

这个中文的话 espeak-ng 要自己编译下加上普通话

**Comment by SWHL** [comment]
最近将 RapidTTS 做了升级，之前版本模型性能有些不够用了。这次集成了 [MeloTTS](https://github.com/myshell-ai/MeloTTS)，后续会持续集成轻量好用的 TTS 模型。敬请期待！

新版直接安装即可使用：`pip install rapidtts`。模型会自动下载。详情参见：[安装说明](https://github.com/RapidAI/RapidTTS/blob/main/docs/installation.md)
